DSCResources/MSFT_xIPAddress/MSFT_xIPAddress.psm1

```diff
@@ -134,48 +130,162 @@ function ValidateProperties
 
     if(!([System.Net.Ipaddress]::TryParse($ip, [ref]0)))
```

Comment:
This should probably be changed to $IPAddress and the line above it $ip = $IPAddress should just be removed.